File: sea-shell-service-immutable/src/main/java/org/jesperancinha/shell/webflux/immutable/SeaShellImmutableApplication.java
package org.jesperancinha.shell.webflux.immutable;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.web.servlet.WebMvcAutoConfiguration;
import org.springframework.boot.autoconfigure.web.servlet.error.ErrorMvcAutoConfiguration;

@SpringBootApplication(exclude = {WebMvcAutoConfiguration.class, ErrorMvcAutoConfiguration.class})
public class SeaShellImmutableApplication {
    public static void main(String[] args) {
        SpringApplication.run(SeaShellImmutableApplication.class, args);
    }

}

File: sea-shell-service-immutable/src/main/kotlin/org/jesperancinha/shell/webflux/immutable/SeaShellImmutableApplication.kt
package org.jesperancinha.shell.webflux.immutable

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.autoconfigure.web.servlet.WebMvcAutoConfiguration
import org.springframework.boot.autoconfigure.web.servlet.error.ErrorMvcAutoConfiguration

@SpringBootApplication(exclude = [WebMvcAutoConfiguration::class, ErrorMvcAutoConfiguration::class])
class SeaShellImmutableApplication

fun main(args: Array<String>) {
    SpringApplication.run(SeaShellImmutableApplication::class.java, *args)
}